xbmc/music/ContextMenus.cpp

```diff
@@ -35,6 +36,22 @@
 namespace CONTEXTMENU
 {
 
+static bool IsMusicDbItem(const CFileItem& item)
+{
+  if (URIUtils::IsMusicDb(item.GetPath()))
```

Comment:
Hmmm, are these comments result of a runtime test or just out of theory? 

I thought parent folders do not have "musicdb" urls and I do not understand why the node type should be checked if the item has no "musicb" url. Doing the latter leads to the problem we're currently facing - namely "set as default" / "clear default" appearing at various wrong places.